src/main/java/modul1/Product.java

```diff
@@ -0,0 +1,65 @@
+package modul1;
+
+public class Product {
+
+    private String id;
+    private String name;
+
+    private String allProducts;
+
+    public Product(String id, String name) {
+    }
```

Comment:
Hier solltest du die Konstruktor Parameter den Klassen Attributen von Product zuweisen

```suggestion
    public Product(String id, String name) {
        this.id = id;
        this.name = name;
    }
```